### ISSUE TITLE
Fluid voiding on server shutdown/startup and not fully stopping upon /stop

### ISSUE THREAD

**Issue by muggelpotato** [Nomi-CEu/Nomi-CEu#656]
### Nomi CEu Version

1.7-alpha-2a

### Launcher

Prism Launcher

### Environment

Dedicated Forge server

### Mode

Expert Mode

### What Happened

Setup:
HOG fed into a rotary heart furnace via AE2 fluid interface on Luv nonuple input hatch.
Fluid Storage = Quantum Tank 1(Fluid locking/voiding enabled), AE2 Fluid storage bus on the back, Quantum tank 1 output wrenched to the side.

Problem 1:
Before scheduled 12:00 server reboot, Rotary heart furnace was fed with HOG and draconium dust (actively processing). Had 64K HOG left at the time. 
*reboot*
When I came back to check, ~4k drac dust was processed but no more HOG was left in the ME System.  
There should be 60K HOG left.
In general I had the suspicion that over time some other fluids got voided as well despite not being used.
Everything else in the savegame is totally fine.

Might not be an issue with the pack itself. Had Problems with server reboot scripts before due to our special server setup.

Problem 2:
After updating the pack and Linux a while ago we also had trouble with stopping the pack completly. 
Basically:
1) stop command
2) backups the last state of savegame folder before complete shutdown
3) backup finishes fine
4) server idles in a semi stopped state
5) Linux kills the process because it takes to long (2-3min)
6) immediately crashes upon next start

I need to replace the savegame folder with the Backup (2)) every time I wanna start the service again.
Didn't change anything else (server scripts etc.) except updating Linux and the pack from v1.6.1a to v1.7-alpha-2a

### Expected Behavior

Not steeling my precious HOG.
Stopping correctly.

### Reproduction Steps

Fluid voiding is highly random.

### Additional Information

I am sorry if that is a clear server setup issue but I'm desperate.

**Comment by rogerwim** [comment]
I am hosting a server on 1.7 alpha 2a and am not having any of these issues myself

**Comment by IntegerLimit** [comment]
Closed as not reproduceable.
